packages/core/services/signers/EosMemoryDevice.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import EventEmitter from 'events'
+import Eos from 'eosjs'
+import ecc from 'eosjs-ecc'
+import ethUtils from 'ethereumjs-util'
+import { eddsa as EdDSA } from 'elliptic'
+
+export default class EosMemoryDevice extends EventEmitter {
+  constructor ({ privateKey }) {
+    super()
+    this.keys = this.createEosKeys(privateKey)
+    Object.freeze(this)
+  }
+
+  createEosKeys (ethereumPrivateKey) {
+    const ethPkBuf = Buffer.from(ethereumPrivateKey, 'hex')
+    if (ethUtils.isValidPrivate(ethPkBuf)) {
+      // Create EOS owner keys
+      const convertedEOSOwnerPrivateKey = ecc.PrivateKey(ethPkBuf).toWif()
+      const convertedEOSOwnerPublicKey = ecc.privateToPublic(convertedEOSOwnerPrivateKey)
+
+      const ec = new EdDSA('ed25519') // ed25519 - preset from library
+      // Create key pair for active key
+      const keyPair = ec.keyFromSecret(ethPkBuf) // hex string, array or Buffer
+      // Create EOS active keys
+      const convertedEOSActivePrivateKey = ecc.PrivateKey(Buffer.from(keyPair.priv().toString('hex'), 'hex')).toWif()
+      const convertedEOSActivePublicKey = ecc.privateToPublic(convertedEOSActivePrivateKey)
+
+      // eslint-disable-next-line
```

Comment:
Please use 
/* eslint-disable no-console */
... consoles ...
/* eslint-enable no-console */

And remove all these logs after that. ;-)